src/pages/project/project.jsx

```diff
@@ -59,6 +60,8 @@ module.exports = React.createClass({
         return console.error('Could not fetch the page');
       }
 
+      platform.trackEvent('Browser Player', 'Open Project', 'Open Project Success', options.uri);
```

Comment:
`options.uri` is the url for the current page e.g. http://beta.webmaker.org/#/player?user=1&project=1